russh/src/client/mod.rs

```diff
@@ -77,6 +77,7 @@ mod session;
 /// It is in charge of multiplexing and keeping track of various channels
 /// that may get opened and closed during the lifetime of an SSH session and
 /// allows sending messages to the server.
+
```

Comment:
Not sure if this newline is intentional ?

Reply:
nope, removed